src/Ui.Playground/GraphQLPlaygroundOptions.cs

```diff
@@ -11,6 +11,16 @@ public class GraphQLPlaygroundOptions {
         /// </summary>
         public PathString GraphQLEndPoint { get; set; } = "/graphql";
 
+        /// <summary>
+        /// The GraphQL Config (as JSON string)
+        /// </summary>
+        public string GraphQLConfig { get; set; } = null;
+
+        /// <summary>
+        /// The GraphQL Playground Settings (as JSON string)
+        /// </summary>
+        public string PlaygroundSettings { get; set; } = null;
```

Comment:
I think I would lean towards this being a Dictionary<string, object> vs just a string.  Then serialize that to JSON.

Reply:
I know that is how the other property is setup, just seems a bit odd to me.

Reply:
Ok, changed to Dictionary now. Using a string smelled a bit strange, that's right.